### PR TITLE
remove Context class

### PR DESCRIPTION
**Pull request by sclassen**

### CODE REVIEW

File: base/rico-core/src/main/java/dev/rico/core/lang/StringPair.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * limitations under the License.
+ */
+package dev.rico.core.lang;
+
+import dev.rico.internal.core.lang.DefaultStringPair;
+
+/**
+ * Defines a key-value pair both of which are strings
+ */
+public interface StringPair extends Tuple<String, String> {
```
Comment (hendrikebbers) [inline base/rico-core/src/main/java/dev/rico/core/lang/StringPair.java:23]: If we name this interface StringPair we maybe should rename Tuple to Pair ?

File: base/rico-core/src/main/java/dev/rico/internal/core/context/ContextManagerImpl.java [hunk 2]
```diff
@@ -45,86 +43,69 @@
...
+            setGlobalAttribute(HOST_ADDRESS_CONTEXT, address.getHostAddress());
         } catch (Exception e) {
             LOG.error("Can not define InetAddress for context!", e);
         }
+    }
 
+    @Override
+    public Subscription setGlobalAttribute(final String name, final String value) {
+        Assert.requireNonNull(name, "name");
+        Assert.requireNonNull(value, "value");
```
Comment (hendrikebbers) [inline base/rico-core/src/main/java/dev/rico/internal/core/context/ContextManagerImpl.java:66]: I would assume that the value could be null

File: base/rico-server/src/main/java/dev/rico/internal/server/context/ContextServerUtil.java [hunk 3]
```diff
@@ -16,38 +16,37 @@
...
-public class ContextServerUtil {
+public final class ContextServerUtil {
 
     private ContextServerUtil() {}
 
-    public static Context createPortTag(final HttpServletRequest request) {
+    public static StringPair createPortTag(final HttpServletRequest request) {
         Assert.requireNonNull(request, "request");
-        return new ContextImpl("port", String.valueOf(request.getServerPort()));
+        return StringPair.of("port", String.valueOf(request.getServerPort()));
```
Comment (hendrikebbers) [inline base/rico-server/src/main/java/dev/rico/internal/server/context/ContextServerUtil.java]: We should create constants for the string values

File: base/rico-core/src/main/java/dev/rico/core/lang/StringPair.java (hunk 1 above)
Comment (sclassen) [reply base/rico-core/src/main/java/dev/rico/core/lang/StringPair.java:23]: yes in a next PR

**Pull request merged**